package.json

```diff
@@ -13,16 +13,17 @@
     "prepare": "husky install",
     "build": "pnpm --recursive build",
     "build-vscode": "pnpm --filter='neo4j-for-vscode...' build",
-    "dev-codemirror": "pnpm --filter='...@neo4j-cypher/react-codemirror-playground' build && pnpm --filter='@neo4j-cypher/react-codemirror-playground' dev",
+    "dev-codemirror": "pnpm --recursive --parallel --filter='@neo4j-cypher/react-codemirror-playground...' dev",
+    "dev-vscode": "pnpm --recursive --parallel --filter='neo4j-for-vscode...' dev",
```

Comment:
This means run `dev` in `neo4j-for-vscode` and all it's dependencies in parallel